Does `make check` fails at mypy `subtest` check?

Steps:
- download/clone the project
- create virtual environment with Py3.9 (done with `uv venv --python 3.9`)
    - `which python3` gives the path to the interpreter within the venv
- install the packages: black. flake8, mypy (done with `uv add black flake8 mypy`)
- run `make check` (or `python3 -m mypy.stubtest more_itertools`)

I get the following output:
```shell
╰─➤  make check
black --check .
All done! ✨ 🍰 ✨
11 files would be left unchanged.
flake8 more_itertools tests
stubtest more_itertools.more more_itertools.recipes
error: more_itertools.more.__all__ is not present in stub
Stub: in file /home/.../Projects/more-itertools-master/more_itertools/more.pyi
MISSING
Runtime: in file /home/.../Projects/more-itertools-master/more_itertools/more.py
<module 'more_itertools.more' from '/home/.../Projects/more-itertools-master/more_itertool...

error: more_itertools.recipes.__all__ is not present in stub
Stub: in file /home/.../Projects/more-itertools-master/more_itertools/recipes.pyi
MISSING
Runtime: in file /home/.../Projects/more-itertools-master/more_itertools/recipes.py
<module 'more_itertools.recipes' from '/home/.../Projects/more-itertools-master/more_itert...

Found 2 errors (checked 2 modules)
make: *** [Makefile:13: check] Error 1
```

This was the reason to comment the subtest stage [here](https://github.com/more-itertools/more-itertools/pull/934/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R12) in the PR #934 .

The output of the `uv pip list`:
```shell
╰─➤  uv pip list             
Package           Version Editable project location
----------------- ------- -----------------------------------------------
black             24.10.0
click             8.1.7
flake8            7.1.1
mccabe            0.7.0
more-itertools    10.5.0  /home/.../Projects/more-itertools-master
mypy              1.14.0
mypy-extensions   1.0.0
packaging         24.2
pathspec          0.12.1
platformdirs      4.3.6
pycodestyle       2.12.1
pyflakes          3.2.0
tomli             2.2.1
typing-extensions 4.12.2
```

Could someone clarify about the reasons and possible ways to fix this?

--- a/requirements/development.txt
+++ b/requirements/development.txt
@@ -2,7 +2,7 @@ black
 coverage
 flake8
 flit
-mypy
+mypy==1.13
 setuptools
 sphinx
 sphinx_rtd_theme

